server/datastore/mysql/software.go

```diff
@@ -957,7 +957,8 @@ func (ds *Datastore) HostsBySoftwareIDs(ctx context.Context, softwareIDs []uint)
 	queryStmt := `
     SELECT 
       h.id,
-      h.hostname
+      h.hostname,
+      if(h.computer_name = '', h.hostname, h.computer_name) display_name
```

Comment:
I may be misunderstanding the change, but I would've expected queries like that to use the new `host_display_names` table instead of replicating the logic to generate the display name?

Reply:
Ah gotcha :+1: